src/main/webapp/app/exercises/shared/exercise/exercise.service.ts

```diff
@@ -72,14 +72,20 @@ export class ExerciseService {
      */
     validateDate(exercise: Exercise) {
         exercise.dueDateError = this.hasDueDateError(exercise);
+        exercise.startDateError = this.hasStartDateError(exercise);
         exercise.assessmentDueDateError = this.hasAssessmentDueDateError(exercise);
 
         exercise.exampleSolutionPublicationDateError = this.hasExampleSolutionPublicationDateError(exercise);
         exercise.exampleSolutionPublicationDateWarning = this.hasExampleSolutionPublicationDateWarning(exercise);
     }
 
+    hasStartDateError(exercise: Exercise) {
```

Comment:
```suggestion
    isInvalidStartDate(exercise: Exercise) {
```
I think that phrasing makes more sense, doesn't it?